### ISSUE TITLE
Non Full screen Widgets

### ISSUE THREAD

**Issue by BigRedJim** [nfm886/skin.cosmic#25]
Hi Heppen

Thanks for the awesome work you have done on the skin so far, it's really coming on well. I wondered if you might consider introducing a feature where you can toggle on/off widgets being full screen and showing information above them so you can have more than one on the screen at the same time? This would be done on a per widget basis. 

To illustrate what I mean please see the below screenshot of how Cosmic handles 2 widgets rows and how Artic Horizon handles it (the information above the widget can be toggled individually for each widget in that skin).

Cosmic 1st Row
![Screenshot (Cosmic1)](https://user-images.githubusercontent.com/47058841/130445532-b8ef4697-723a-4063-b32b-3455f30c194e.png)
Cosmic 2nd Row
![Screenshot (Cosmic2)](https://user-images.githubusercontent.com/47058841/130445537-b37a7e87-3ed7-47c7-be8e-7a0918196e14.png)
AH
![Screenshot (AH)](https://user-images.githubusercontent.com/47058841/130445478-cf4ea307-4de2-49c8-a0b8-089a9fda3e2d.png)

As a secondary matter while you are looking at the settings you have for each individual widget I also think being able to hide watched items on a per widget basis would be really useful as well, I've not seen that in another skin though so not sure if it's possible or not.

Appreciate your time,  cheers.

**Comment by icpryde** [comment]
Wouldn’t the double widget though defeat the purpose of the trailer feature ? Unless i guess if you plan to have the trailer/poster feature off than this makes sense unless I’m missing something

**Comment by BigRedJim** [comment]
> Wouldn’t the double widget though defeat the purpose of the trailer feature ? Unless i guess if you plan to have the trailer/poster feature off than this makes sense unless I’m missing something, but this would get in the way of the main UI feature.

Yes I have that turned off so I hadn't even thought about it, what you say makes sense but as the auto trailer is able to be turned on/off then I still think this would be a good option for those that choose to have it off.

**Comment by rijsab** [comment]
In my opinion this is indeed desirable, especially for continu watching sections were you don't want to play the trailers. I have a hoke section were I have my continue watching widgets for local library, Netdlix and Prime Video and were I would like to turn off the trailers for this section only and keep using for other sections. But nfm886 already made great progress on designing a new skin with cool features and smooth design.

**Comment by nfm886** [comment]
I will consider this as an option. Will be back to this ticket in free time :)

**Comment by heppen7** [comment]
Overdone. Room for new issues after new update.
